script/libtest.bash

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This is a library rather than a file intended to be directly executable.  So we should probably remove the `#!` line entirely.

Reply:
I agree.